cpp/dolfinx/mesh/utils.h

```diff
@@ -41,35 +42,39 @@ enum class GhostMode : int
 
 namespace impl
 {
-/// Re-order an adjacency list of fixed degree
+/// @brief Re-order the nodes a fixed-degree adjacency list.
```

Comment:
@garth-wells  Did you mean
"Re-order the nodes of a fixed-degree adjacency list"